oremda/clients/singularity/client.py

```diff
@@ -96,3 +98,19 @@ def resolve_path(self, path):
         for p in paths_tried:
             msg += f"  {p}\n"
         raise Exception(msg)
+
+    def images(self, organization: str = None) -> List[ImageBase]:
```

Comment:
The return type here could be `List[SingularityImage]` instead, right? Then I think we also wouldn't need to import `ImageBase`.

Reply:
Yup, fixed